index.js

```diff
@@ -59,6 +60,12 @@ module.exports = function (opts) {
     'text/plain',
   ];
 
+  // default xml formTypes
+  var xmlTypes = [
```

Comment:
May I suggest adding ``application/soap+xml`` to the list?

Reply:
agree to this